tests/5.1/default/test_default_firstprivate_task.c

```diff
@@ -0,0 +1,58 @@
+//--------------- test_default_firstprivate_task.c -------------------------//
```

Comment:
change header and name of test to test_task_default_firstprivate.c, that better matches our naming schema!